azure-kusto-data/azure/kusto/data/version.py

```diff
@@ -1,2 +1,4 @@
 """ Version file. """
+# review: why do you have a separate file for the version? Why can't you just
+# inline this in __init__.py?
```

Comment:
I need it at kusto_client

Reply:
I would actually put all files in the `data/` folder into a single `data.py` and drop the subpackage all together.